frontends/web/src/routes/account/account.tsx

```diff
@@ -312,8 +317,12 @@ class Account extends Component<Props, State> {
                     {status.synced && this.dataLoaded() && isBitcoinBased(account.coinCode) && <HeadersSync coinCode={account.coinCode} />}
                     <div className="innerContainer scrollableContainer">
                         <div className="content padded">
-                            <Status dismissable={`info-${code}`} type="info" className="m-bottom-default">
-                                {t(`account.info.${code}`, { defaultValue: '' })}
+                            <Status
+                                className="m-bottom-default"
+                                hidden={!coinCodeInfo}
```

Comment:
So, this status is now hidden if a translation is missing? unless I don't understand what it does exactly:

```js
const coinCodeInfo: string = t(`account.info.${code}`, { defaultValue: '' });
```

My guess is `coinCodeInfo` is empty string if either a translation is missing or there's no such key. But how can it be that a key is missing and why should it impact the hidden prop of the status? I'm confused.

Reply:
> Some situations the whole status component was wrapped in a
condition, others only the content of the status component was
in a condition. Now the condition can be passed in the hidden
prop.

I should update the commit description.

> So, this status is now hidden if a translation is missing? 

it already was doing that before, but by checking if the child is truthy else return null. I'll remove that logic as it is not needed anymore as it can be controlled with the hidden props now.

Reply:
I still don't understand how this works:

```ts
const coinCodeInfo: string = t(`account.info.${code}`, { defaultValue: '' });
hidden={!coinCodeInfo}
```

Why would a translated string impact the `hidden` attribute value? It doesn't sound like the right condition to me.

Reply:
I keep the logic, before this change the condition was it rendered null if children are falsy `!childrenList[0]` so for example an empty string. With `!childrenList[0]` here, (this is also what broke in PreactX and proably will in React too)

https://github.com/digitalbitbox/bitbox-wallet-app/blob/0b03df07ea60ba491bd5eab52b1b43596c09284b/frontends/web/src/components/status/status.tsx#L94

Reply:
> Why would a translated string impact the hidden attribute value? It doesn't sound like the right condition to me.

I agree, but need to investigate…

Reply:
Ok this is pretty old and only used when the legacy setting split accounts is enabled.

The info is only shown on the different Bitcoin address types, if the translation is missing (and defaults to an empty string) then no status component is rendered.

https://github.com/digitalbitbox/bitbox-wallet-app/blob/0b03df07ea60ba491bd5eab52b1b43596c09284b/frontends/web/src/locales/en/app.json#L8-L15

Reply:
Don't you find this logic a bit concerning? Is it possible to change to something more reasonable like this:

```ts
hidden={legacySplitIsEnabled}
```

I'm sure frontend can get this setting from somewhere.

Reply:
added  WithCoinTypeInfo 
```js
const WithCoinTypeInfo = [
    'btc-p2pkh',
    'btc-p2wpkh',

];
```
squashed force pushed in https://github.com/digitalbitbox/bitbox-wallet-app/compare/6108003c9fdb5b3e4bbbe0d8a9f668c07d4ac366..4663377c71c832cb9c7a73fa3733b92186e730fe

also addressed the other comment, which I cannot comment on anymore.

Reply:
:pray: but please add a short comment to explain what `WithCoinTypeInfo` is for.